src/SourceGenerators/Uno.UI.SourceGenerators/BindableTypeProviders/DependencyObjectAvailabilityGenerator.cs

```diff
@@ -141,6 +148,51 @@ internal void Generate(GeneratorExecutionContext context)
 				}
 			}
 
+			private HashSet<INamedTypeSymbol> FindAdditionalLinkerHints(INamedTypeSymbol? additionalLinkerHintAttributeSymbol, GeneratorExecutionContext context)
+			{
+				HashSet<INamedTypeSymbol> types = new();
```

Comment:
It might be good to pass `SymbolEqualityComparer.Default` to the constructor that takes an `IEqualityComparer<T>`